From e99d1768b77e77182d5e7ac9c335c7d8cbd88963 Mon Sep 17 00:00:00 2001
From: Jonathan Bell <jonathan@raspberrypi.com>
Date: Mon, 20 May 2024 14:45:37 +0100
Subject: [PATCH 1096/1135] DTS: bcm2712: don't assume L1 sub-state support at
 chip level

L1 sub-states require the connection of the CLKREQ# auxiliary signal
which is a PCB-level decision, and also depends on the vagrancies of
adapters/interposers/slots that may exist between the chip and the
endpoint.

If this parameter is present, and the RC driver sees an L1 substate
capability in the endpoint, then the result is a broken link if CLKREQ#
isn't end-to-end connected - regardless of the state of the L1.x enable
bits in the RC.

Board-specific methods (HAT+ overlay, board DTB) should define this
property if required.

Signed-off-by: Jonathan Bell <jonathan@raspberrypi.com>
---
 arch/arm/boot/dts/broadcom/bcm2712.dtsi | 2 --
 1 file changed, 2 deletions(-)

--- a/arch/arm/boot/dts/broadcom/bcm2712.dtsi
+++ b/arch/arm/boot/dts/broadcom/bcm2712.dtsi
@@ -1063,7 +1063,6 @@
 				      0x00 0x00000000
 				      0x10 0x00000000>;
 
-			brcm,enable-l1ss;
 			status = "disabled";
 		};
 
@@ -1124,7 +1123,6 @@
 				      0x00 0x00000000
 				      0x10 0x00000000>;
 
-			brcm,enable-l1ss;
 			status = "disabled";
 		};
 
